mesos-swarm-marathon/azuredeploy.json

```diff
@@ -0,0 +1,401 @@
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for the Virtual Machine."
+      }
+    },
+    "dnsNameForJumpboxPublicIP": {
+      "type": "string",
+      "metadata": {
+        "description": "Unique DNS Name for the Public IP used to access the Jumpbox."
+      }
+    },
+    "dnsNameForContainerServicePublicIP": {
+      "type": "string",
+      "metadata": {
+        "description": "Unique DNS Name for the Public IP used to access the Virtual Machine."
+      }
+    },
+    "agentCount": {
+      "type": "int",
+      "defaultValue": 0,
+      "metadata": {
+        "description": "The number of mesos masters for the cluster."
+      }
+    },
+    "masterCount": {
+      "type": "int",
+      "defaultValue": 1,
+      "allowedValues": [
+        1,
+        3,
+        5
+      ],
+      "metadata": {
+        "description": "The number of mesos masters for the cluster."
+      }
+    },
+    "jumpboxCount": {
+      "type": "int",
+      "defaultValue": 0,
+      "allowedValues": [
+        0,
+        1
+      ],
+      "metadata": {
+        "description": "The number of mesos masters for the cluster."
+      }
+    },
+    "masterConfiguration": {
+      "type": "string",
+      "defaultValue": "masters-are-agents",
+      "allowedValues": [
+        "masters-are-agents",
+        "masters-are-not-agents"
+      ],
+      "metadata": {
+        "description": "Specify whether masters should act as agents or not."
+      }
+    },
+    "agentVMSize": {
+      "type": "string",
+      "defaultValue": "Standard_D1",
+      "allowedValues": [
+        "Standard_D1",
+        "Standard_D2",
+        "Standard_D3",
+        "Standard_D4",
+        "Standard_D11",
+        "Standard_D12",
+        "Standard_D13",
+        "Standard_D14"
+      ],
+      "metadata": {
+        "description": "The size of the Virtual Machine."
+      }
+    },
+    "masterVMSize": {
+      "type": "string",
+      "defaultValue": "Standard_D1",
+      "allowedValues": [
+        "Standard_D1",
+        "Standard_D2",
+        "Standard_D3",
+        "Standard_D4",
+        "Standard_D11",
+        "Standard_D12",
+        "Standard_D13",
+        "Standard_D14"
+      ],
+      "metadata": {
+        "description": "The size of the Virtual Machine."
+      }
+    },
+    "jumpboxVMSize": {
+      "type": "string",
+      "defaultValue": "Standard_D2",
+      "allowedValues": [
+        "Standard_D1",
+        "Standard_D2",
+        "Standard_D3",
+        "Standard_D4",
+        "Standard_D11",
+        "Standard_D12",
+        "Standard_D13",
+        "Standard_D14"
+      ],
+      "metadata": {
+        "description": "The size of the Virtual Machine."
+      }
+    },
+    "clusterPrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "The prefix to identify the cluster."
+      }
+    }
+  },
+  "variables": {
+    "adminUsername": "azureuser",
```

Comment:
Please let the user choose the username so that they remember and also as a security best practice